tsconfig.json

```diff
@@ -6,9 +6,10 @@
     "esModuleInterop": true, /* Emit additional JavaScript to ease support for importing CommonJS modules. This enables 'allowSyntheticDefaultImports' for type compatibility. */
     "forceConsistentCasingInFileNames": true, /* Ensure that casing is correct in imports. */
     "strict": true, /* Enable all strict type-checking options. */
+    "skipLibCheck": true,
```

Comment:
We need `skipLibCheck: true` turned off for `ts-reset` - was there a reason you turned it on?

Reply:
hey Matt, big fan of your work! I turned it on because for the test to work, I had to include the `/// <reference lib="webworker" />`

but with skipLibCheck off, I got type errors with the built-in webworker types

> We need skipLibCheck: true turned off for ts-reset

ohh why do we need this?

Reply:
the kinda errors I'm getting without `skipLibCheck:true`:
```
 node_modules/.pnpm/typescript@5.5.4/node_modules/typescript/lib/lib.webworker.d.ts:23:1
    23 interface AddEventListenerOptions extends EventListenerOptions {
       ~~~~~~~~~
    Conflicts are in this file.

node_modules/.pnpm/typescript@5.5.4/node_modules/typescript/lib/lib.dom.d.ts:3632:5 - error TS2374: Duplicate index signature for type 'number'.

3632     [index: number]: CSSNumericValue;
```